src/quo2/components/notifications/activity_log/view.cljs

```diff
@@ -165,6 +165,9 @@
     :as   props}]
   [rn/view
    {:accessibility-label :activity
+    ;; don't propagate touch events to child views (message component)
+    ;; https://github.com/status-im/status-mobile/issues/16706
+    :pointer-events      :none
```

Comment:
@Parveshdhull, I'm afraid we can't add `:pointer-events :none` to the top-level view, otherwise button events don't work (e.g. Accept/Decline buttons for contact requests). I tested this case just now.